src/main/java/moze_intel/projecte/emc/ComplexGraphMapper.java

```diff
@@ -55,7 +55,7 @@ public ComplexGraphMapper(IValueArithmetic arithmetic) {
 						debugFormat("Set value for %s to %d because %d/%d Conversions solved\n", something.toString(), minValue, getNoDependencyConversionCountFor(something), getConversionsFor(something).size());
 					}
 				}
-				System.out.format("lookat: %d solvable: %d\n", lookAt.size(), solvableThings.size());
+				System.out.format("lookat: %d solvable: %d%n", lookAt.size(), solvableThings.size());
```

Comment:
Why do you use %n here? isn't that the evil format code that writes into variables instead of reading?